source/administration/command-line-tools.rst

```diff
@@ -2674,3 +2674,16 @@ CLI Documentation:
       -version                          Display the current of the Mattermost platform
 
       -help                             Displays this help page
+
+
+Troubleshooting
+^^^^^^^^^^^^^^^^^
+
+Executing a command hangs and doesn't complete
+------------------------------------------------
+
+If you have Bleve search indexing enabled, temporarily disable it in **System Console > Experimental > Bleve** and run the command again. You can also optionally use a new `mmctl Command Line Tool <https://docs.mattermost.com/administration/mmctl-cli-tool.html>`_ instead.
```

Comment:
```suggestion
If you have Bleve search indexing enabled, temporarily disable it in **System Console > Experimental > Bleve** and run the command again. You can also optionally use the new `mmctl Command Line Tool <https://docs.mattermost.com/administration/mmctl-cli-tool.html>`_ instead.
```

Both work, but as we're talking about a specific tool **the** might be more appropriate?